if (fn === null) throw new Error("Callback was already called.");

run my site on ghost docker images version 0.11.4, always crash around 1~2 months, the logs indicate throw error inside amperize node_modules, post here for reference.

docker's node version:
 4.7.2

docker's log:
66.249.65.234 - - [18/Jan/2017:04:37:52 +0000] "GET /2015/11/03/wei-ruan-su-fei-bi-ji-ben-dai-du-xian/amp/ HTTP/1.1" 200 - "-" "Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.96 Mobile Safari/537.36 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)"
/usr/src/ghost/node_modules/amperize/node_modules/async/dist/async.js:844
            if (fn === null) throw new Error("Callback was already called.");
                             ^

Error: Callback was already called.
    at /usr/src/ghost/node_modules/amperize/node_modules/async/dist/async.js:844:36
    at /usr/src/ghost/node_modules/amperize/node_modules/async/dist/async.js:2478:17
    at close (/usr/src/ghost/node_modules/amperize/lib/amperize.js:109:16)
    at enter (/usr/src/ghost/node_modules/amperize/lib/amperize.js:118:16)
    at IncomingMessage.<anonymous> (/usr/src/ghost/node_modules/amperize/lib/amperize.js:199:24)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:923:12)
    at nextTickCallbackWith2Args (node.js:458:9)
    at process._tickDomainCallback (node.js:413:17)

npm info ghost@0.11.4 Failed to exec start script
npm ERR! Linux 3.16.0-4-amd64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.7.2
npm ERR! npm  v2.15.11
npm ERR! code ELIFECYCLE
npm ERR! ghost@0.11.4 start: `node index`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the ghost@0.11.4 start script 'node index'.
npm ERR! This is most likely a problem with the ghost package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs ghost
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! 
npm ERR!     npm owner ls ghost
npm ERR! There is likely additional logging output above.
npm ERR! Linux 3.16.0-4-amd64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.7.2
npm ERR! npm  v2.15.11
npm ERR! path npm-debug.log.896050243
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall open

npm ERR! Error: EACCES: permission denied, open 'npm-debug.log.896050243'
npm ERR!     at Error (native)
npm ERR!  { [Error: EACCES: permission denied, open 'npm-debug.log.896050243']
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'open',
npm ERR!   path: 'npm-debug.log.896050243' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/src/ghost/npm-debug.log

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "node": "^4.2.0 || ^6.9.0"
   },
   "dependencies": {
-    "amperize": "1.0.0",
+    "amperize": "0.3.4",
     "archiver": "1.3.0",
     "bcryptjs": "2.3.0",
     "bluebird": "3.4.7",

